models/palm_detection_mediapipe/demo.py

```diff
@@ -106,12 +111,18 @@ def visualize(image, score, palm_box, palm_landmarks, fps=None):
 
             # Inference
             tm.start()
-            score, palm_box, palm_landmarks = model.infer(frame)
+            results = model.infer(frame)
             tm.stop()
 
             # Draw results on the input image
-            if score is not None and palm_box is not None and palm_landmarks is not None:
-                frame = visualize(frame, score, palm_box, palm_landmarks, fps=tm.getFPS())
+            if results is not None:
```

Comment:
if `results` is an empty array, we do not need this if condition and the for loop stops.